tracer/tracer.go

```diff
@@ -0,0 +1,36 @@
+package tracer
```

Comment:
I'm not sure we should be doing tracing yet. I'd prefer us to figure out logs & metrics first.

Reply:
If we're not doing tracing yet, then there might be no need for New Relic at the moment.

Reply:
I don't think so -- Error tracking, Instrumentation.